### PR TITLE
Update rcudbinfo.md

### PR DESCRIPTION
**Pull request by jshum2479**
Minor update to clarify password fields

### CODE REVIEW

**Comment by jshum2479** [comment]
Minor update to clarify the password fields

**Review by rakillen — APPROVED** [review]

File: site/rcuinfo.md [hunk 1]
```diff
@@ -32,8 +34,10 @@ domainInfo:
         # Optional rcu_db_user for creating RCU schema if -run_rcu flag is specified. Default user is SYS if not specified
         # This user must have SYSDBA privilege and this is the equivalent of -dbUser in the RCU utility
         rcu_db_user: superuser
-        rcu_schema_password : xxxxx
-        rcu_admin_password : xxxx
+        rcu_schema_password : <rcu schema password, will be prompted if not specify>
+        rcu_admin_password : <database admin password is required only when you specify -run_rcu flag, will be prompted
+         if 
+        not specify>
```
Comment (ddsharpe) [inline site/rcuinfo.md:40]: specify -> specified

**Review by ddsharpe — CHANGES_REQUESTED** [review]

**Review by rakillen — APPROVED** [review]

File: site/rcuinfo.md [hunk 2]
```diff
@@ -32,8 +34,10 @@ domainInfo:
         # Optional rcu_db_user for creating RCU schema if -run_rcu flag is specified. Default user is SYS if not specified
         # This user must have SYSDBA privilege and this is the equivalent of -dbUser in the RCU utility
         rcu_db_user: superuser
-        rcu_schema_password : xxxxx
-        rcu_admin_password : xxxx
+        rcu_schema_password : <rcu schema password, will be prompted if not specify>
```
[the remark this answers is not in the document]
Comment (jshum2479) [reply site/rcuinfo.md:37]: done

File: site/rcuinfo.md (hunk 1 above)
Comment (jshum2479) [reply site/rcuinfo.md:40]: done

**Review by ddsharpe — APPROVED** [review]

**Pull request merged**